src/components/terms-and-conditions.tsx

```diff
@@ -40,6 +40,13 @@ export const TermsAndConditions = ({
           ref={field.ref}
           name={field.name}
           checked={field.value}
+          onChange={() => {
+            if (!field.value) {
+              handleshowPopupChange();
+            } else {
+              setValue(field.name, !field.value);
+            }
```

Comment:
Ackchyually, you can use ternary operator to make it more concise.

Reply:
ternary operator my beloved 😍

Reply:
> Ackchyually, you can use ternary operator to make it more concise.

🤓